core/lib/config/src/configs/eth_sender.rs

```diff
@@ -135,6 +136,9 @@ pub struct GasAdjusterConfig {
     pub poll_period: u64,
     /// Max number of l1 gas price that is allowed to be used in state keeper.
     pub max_l1_gas_price: Option<u64>,
+    /// Number of blocks collected by GasAdjuster from which blob_base_fee median is taken
+    #[serde(default = "GasAdjusterConfig::default_max_blob_base_fee_samples")]
+    pub max_blob_base_fee_samples: usize,
```

Comment:
the name was quite confusing (I didn't understand it until I read the comment).

My suggestion would be:
num_samples_for_blob_base_fee_estimate